packages/calypso-e2e/src/lib/components/sidebar-component.ts

```diff
@@ -82,13 +82,9 @@ export class SidebarComponent {
 		}
 
 		// Verify the expected item or subitem is selected.
-		await Promise.all( [
-			this.page.waitForSelector( selectedMenuItem, {
-				timeout: 10000,
-				state: 'attached',
-			} ),
-			this.page.waitForSelector( '.focus-content' ),
-		] );
+		await this.page.waitForSelector( '.focus-content' );
+		const locator = this.page.locator( selectedMenuItem );
+		await locator.elementHandle();
```

Comment:
Do you think the following equivalent would be easier to follow?
```suggestion
		await locator.waitFor();
```

Reply:
I don't particularly see the difference between the two, though do keep in mind that we're on Playwright 1.14 and the Locator API in 1.14 does not support  `waitFor`.

Reply:
We've upgraded to Playwright 1.16 with #59159 (although with a slight compromise), so I can look into using `waitFor` instead!

Reply:
Changed in 923d20f8ca25a27b0bc5a93b11cf845f7cddd6c1.